doc/api/child_process.md

```diff
@@ -293,6 +293,12 @@ added: v0.1.90
 * `options` {Object}
   * `cwd` {String} Current working directory of the child process
   * `env` {Object} Environment key-value pairs
+  * `argv0` {String} Explicitly set the value of `argv[0]` sent to the child
+    process. This will be set to `command` if not specified. *Note:* Certain
```

Comment:
I would move each of the `*Note*` instances out of the bullet list and into their own paragraphs for consistency with *Note* blocks used elsewhere in the documentation. Once I get through all of the documentation to make the *Note* blocks consistent I'm going to be working on improving how those are displayed in the rendered documentation.